sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -647,6 +647,43 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     PathAccessConditions AccessConditions;
   };
 
+  /**
+   * @brief Optional parameters for PathClient::SetAccessControlRecursive
+   */
+  struct SetDirectoryAccessControlRecursiveOptions
+  {
+    /**
+     * @brief Context for cancelling long running operations.
+     */
+    Azure::Core::Context Context;
+
+    /**
+     * @brief When performing setAccessControlRecursive on a directory, the number of paths that
+     *        are processed with each invocation is limited.  If the number of paths to be processed
+     *        exceeds this limit, a continuation token is returned in this response header.  When a
+     *        continuation token is returned in the response, it must be specified in a subsequent
+     *        invocation of the setAccessControlRecursive operation to continue the
+     *        setAccessControlRecursive operation on the directory.
+     */
+    Azure::Core::Nullable<std::string> Continuation;
```

Comment:
We still need to finalize name of this variable. Create an issue to track that. We can merge this PR.